apollo-router/src/plugins/telemetry/tracing/apollo_telemetry.rs

```diff
@@ -703,7 +710,10 @@ impl SpanExporter for Exporter {
         // We may get spans that simply don't complete. These need to be cleaned up after a period. It's the price of using ftv1.
         let mut traces: Vec<(String, proto::reports::Trace)> = Vec::new();
         for span in batch {
-            if span.attributes.get(&APOLLO_PRIVATE_REQUEST).is_some()
+            if span
+                .attributes
+                .iter()
+                .any(|attr| attr.key == APOLLO_PRIVATE_REQUEST)
```

Comment:
This looks like you've found and fixed a bug. Thanks!

Reply:
@jtescher Could you confirm my assumption? We'd like to merge your PR, but this is the one thing we need to be sure we understand.
I believe you have changed this from the previous `get` because there are multiple attributes which might have the key `APOLLO_PRIVATE_REQUEST`, but would appreciate your confirmation.

Reply:
yeah `span.attributes` is now an un-deduped vec, so all need to be checked in `0.21`